modules/client/src/addresslistVoting/internal/client/estimation.ts

```diff
@@ -87,13 +93,20 @@ export class AddresslistVotingClientEstimation extends ClientCore
       throw new NoProviderError();
     }
 
+    if (!isProposalId(params.proposalId)) {
+      throw new InvalidProposalIdError();
+    }
```

Comment:
Redundant if decodeProposalId already does it

Reply:
removed